include/eve/module/math/constant/cbrt_pi.hpp

```diff
@@ -0,0 +1,77 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/module/core.hpp>
+
+namespace eve
+{
+  //================================================================================================
+  //! @addtogroup core
+  //! @{
+  //! @var cbrt_pi
+  //!
+  //! @brief Callable object computing the cbrt_pi constant value.
```

Comment:
can you add a small latex formula. People may not have any idea what cbrt_pi means, showing the actual value approx. could be nice